light_curves/multiband_lc.ipynb

```diff
@@ -137,14 +137,14 @@
     "#get_lyu_sample(coords, labels)  #z32022ApJ...927..227L\n",
     "#get_lopeznavas_sample(coords, labels)  #2022MNRAS.513L..57L\n",
     "#get_hon_sample(coords, labels)  #2022MNRAS.511...54H\n",
-    "#get_yang_sample(coords, labels)   #2018ApJ...862..109Y\n",
+    "get_yang_sample(coords, labels)   #2018ApJ...862..109Y\n",
     "\n",
     "#now get some \"normal\" QSOs for use in the classifier\n",
     "#there are ~500K of these, so choose the number based on\n",
     "#a balance between speed of running the light curves and whatever \n",
     "#the ML algorithms would like to have\n",
     "\n",
-    "num_normal_QSO = 50\n",
+    "num_normal_QSO = 0\n",
```

Comment:
Seems more natural to comment this out rather than set it to 0. 

Related question: What sample do we want to make the default in this cell? It would be good to pick one and then leave it.

Reply:
I guess default so far has been Yang et al., sample.  I will change to be that